languages/kotlin/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,41 @@
+# What's next?
+
+## Submitting your solution
+
+_Hint_: Use `Code --> Reformat` in [IntelliJ IDEA][intellij-idea-ic] to automatically format your code.
+
+## Entry point
+
+To run your Kotlin program you need to define so-called entry point: function with name `main` with or without arguments. However, in Exercism course we are using automatic tests that are using other functions defined in exercises.
+
+Entry point with arguments (you can use them for building CLI applications):
+
+```kotlin
+fun main(args: Array<String>) {
+    println("Hello, Exercism!")
+    println("Program args are: $args")
+}
+```
+
+Or ignore arguments completely:
+
+```kotlin
+fun main() {
+    println("Hello, Exercism!")
+}
+```
+
+## `Unit` return type
```

Comment:
This is nice content! I feel that this might be given precedence (as in: listed before) the `## Entry point` section.

Reply:
Sorry, didn't get what do you mean here.